### PR TITLE
Solved: url #2

### PR DESCRIPTION
**Pull request by TeoRamosKees**
In this modification, I've added max-w-[calc(100%-3rem)] to limit the width of the URL text to the available space, and overflow-hidden whitespace-nowrap overflow-ellipsis to handle overflow by hiding the excess text and showing an ellipsis. 
#2

### CODE REVIEW

**Comment by PriyanshuPz** [comment]
- [ ] https://vercel.live/link/quine-checkcup-iazcuzbjj-priyanshu-vermas-projects.vercel.app?page=%2F%3FvercelThreadId%3De0sEO&via=in-app-copy-link

Fix the thread to merge the branch @TeoRamosKees

File: components/site-card.tsx [hunk 1]
```diff
@@ -27,7 +27,9 @@ function SiteCard({
             className="w-6 h-6"
             alt={`Favicon of ${url}`}
           />
-          <p className="leading-6 pl-1 font-medium text-lg">{url}</p>
+          <p className="leading-6 pl-1 font-medium text-lg max-w-[calc(100%-3rem)] overflow-hidden whitespace-nowrap overflow-ellipsis">
```
Comment (PriyanshuPz) [inline components/site-card.tsx]: Fix overflow in all screens

**Review by PriyanshuPz — CHANGES_REQUESTED** [review]

File: components/site-card.tsx (hunk 1 above)
Comment (TeoRamosKees) [reply components/site-card.tsx]: I ve now added the feature on "site-list" component, i think its now solved, let me know!

Comment (PriyanshuPz) [reply components/site-card.tsx]: TRY THIS:
add: `overflow-hidden` in the parent `div`.
remove classNames from the `p` tag

**Review by PriyanshuPz — CHANGES_REQUESTED** [review]
TRY THIS:
add: `overflow-hidden` in the parent `div`.
remove classNames from the `p` tag

File: components/site-card.tsx (hunk 1 above)
Comment (TeoRamosKees) [reply components/site-card.tsx]: Done, i've also added the overflow-ellipsis utility class that is used to specify that content should be truncated with an ellipsis (...) if it overflows. The overflow-hidden class hides any content that overflows the container's boundaries, and does not give further information about the content, that's why i've also added the overflow-ellipsis.

**Review by PriyanshuPz — CHANGES_REQUESTED** [review]
Broo keep old classNames of `p` tag as it is. LAST TIME...

I said you to remove styling you added.
Try again you are learning from mistakes.

**Comment by TeoRamosKees** [comment]
I gave old classNames to p tags in both "site-list" and "site-card" components

**Review by PriyanshuPz — APPROVED** [review]

**Pull request merged**